hbogolib/handlereu.py

```diff
@@ -184,7 +184,7 @@ def init_api(self, country):
             'Accept-Encoding': ''
         }
 
-    def setup(self, country):
+    def setup(self, country, forceeng=False):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [HbogoHandler_eu.setup is too complex (19)](https://app.codacy.com/manual/arvvoid/plugin.video.hbogoeu/pullRequest?prid=4940571)